database/engine.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+from settings import settings
+
+SQLALCHEMY_DATABASE_URL = settings.DATABASE_URL
+
+connect_args = {"check_same_thread": False}
+engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args=connect_args)
```

Comment:
The `connect_args` with `check_same_thread: False` is specific to SQLite databases. Ensure that this configuration is appropriate for your database setup. If you plan to use a different database, you may need to adjust or remove this argument.